xml/cap_depl_install_production.xml

```diff
@@ -915,51 +917,80 @@ suse/uaa      2.8.0    A Helm chart for SUSE UAA
 &prompt.user;helm upgrade --recreate-pods <replaceable>susecf-console</replaceable> suse/console \
  --values scf-config-values.yaml
 </screen>
-
-<para>
-</para>
-    
-
-<!-- helm rollback does not work, so upgrades are one-way for now
--->
 </sect1>
+
 <sect1 xml:id="sec.cap.ha-prod">
   <title>Example &ha; Configuration</title>
   <para>
 This example &ha; configuration needs two separate configuration files, one for 
-UAA and one for SCF. The first example is for UAA, 
-<filename>uaa-sizing.yaml</filename>.
+UAA and one for SCF. It may be helpful to read the comments in the 
+<literal>sizing:</literal> section in the default
+<filename>values.yaml</filename> file for each namespace, which describes which
+roles can be scaled and the scaling options for each role. One way is to use 
+<command>helm inspect</command> to read the <literal>sizing:</literal> section 
+in your installed charts:
+</para>
+
+<screen>
+&prompt.user;helm inspect suse/uaa | less +/sizing:
+&prompt.user;helm inspect suse/scf | less +/sizing:
+</screen>
+
+<para>
+Another way is to download a copy of the &helm; charts, 
+unpack them, and then use Perl to extract the information for each role from the 
+<literal>sizing:</literal> section. The following example is for UAA.
+You may see a few non-relevant lines from the sections preceding and following 
+the <literal>sizing:</literal> section. In the following example these are lines 
+29-53, and lines 226-245.
 </para>
+
+<screen>
+&prompt.user;helm fetch suse/uaa
+&prompt.user;tar xzvf cf-2.8.0.tgz
```

Comment:
How did you fetch _uaa_ and extract _cf_? Shouldn't those match?

Reply:
Magic!